### PR TITLE
Bump plugins and jenkins version for build jenkins

### PR DESCRIPTION
**Pull request by michaelyoungstrom**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/jenkins_build/defaults/main.yml [hunk 1]
```diff
@@ -202,13 +205,13 @@ build_jenkins_plugins_list:
...
   - name: 'script-security'
-    version: '1.55'
+    version: '1.56'
     group: 'org.jenkins-ci.plugins'
   - name: 'shiningpanda'
     version: '0.23'
     group: 'org.jenkins-ci.plugins'
   - name: 'slack'
-    version: '2.2'
+    version: '2.20'
```
Comment (jmbowman) [inline playbooks/roles/jenkins_build/defaults/main.yml]: 2.21 was released 8 hours ago, although it doesn't look like there are any critical updates in it.

File: playbooks/roles/jenkins_build/defaults/main.yml [hunk 2]
```diff
@@ -256,7 +259,7 @@ build_jenkins_plugins_list:
     version: '1.3.1'
     group: 'org.jenkins-ci.plugins'
   - name: 'workflow-cps'
-    version: '2.64'
+    version: '2.65'
```
Comment (jmbowman) [inline playbooks/roles/jenkins_build/defaults/main.yml]: 2.66 was released yesterday with a fix or two we probably want.

File: playbooks/roles/jenkins_build/defaults/main.yml [hunk 3]
```diff
@@ -202,13 +205,13 @@ build_jenkins_plugins_list:
     version: '1.0'
     group: 'org.jenkins-ci.plugins'
   - name: 'script-security'
-    version: '1.55'
+    version: '1.56'
```
Comment (jmbowman) [inline playbooks/roles/jenkins_build/defaults/main.yml]: 1.57 was released yesterday, although the changes look minor.

**Review by jmbowman — COMMENTED** [review]
Figures, go to upgrade something and they update it again within a day...

**Comment by michaelyoungstrom** [comment]
@jmbowman  Bumped them and deployed on test jenkins

**Review by jmbowman — APPROVED** [review]

**Pull request merged**